crates/holochain_p2p/src/spawn/actor.rs

```diff
@@ -765,7 +765,14 @@ impl kitsune_p2p::event::KitsuneP2pEventHandler for HolochainP2pActor {
                 self.handle_incoming_must_get_agent_activity(space, to_agent, agent, filter)
             }
             crate::wire::WireMessage::ValidationReceipt { receipt } => {
-                self.handle_incoming_validation_receipt(space, to_agent, receipt)
+                // TODO This shouldn't be removed immediately because existing conductors will will be sending these messages.
+                //      Once this change has been back-ported, it can be removed on develop.
+                warn!("Got a validation receipt in a call, this is deprecated and will be removed in a future release.");
```

Comment:
Maintaining compatibility, mainly for the 0.2 series. I'm happy to remove this completely on `develop`.

Reply:
This issue has been forced by the batching change, there is no longer any compatibility with the previous version